### DISCUSSION TITLE
robots.txt shown not to work propertly

### DISCUSSION THREAD

**Discussion by diderich** [YetiForceCompany/YetiForceCRM#15670] [How to] [answered]
When looking at my configuration, it shows that the `robots.txt` is not configured correctly.
![q](https://user-images.githubusercontent.com/10230144/151325356-365db0da-cb39-442f-a547-61be5971e62e.png)

What does that mean? My `robots.txt` file looks as follows:
![r](https://user-images.githubusercontent.com/10230144/151325658-c021b542-4a84-4a43-9b74-44bb48f552cc.png)

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
Check using url e.g. https://gitdeveloper.yetiforce.com/robots.txt

**Reply by diderich** [reply]
URL works, ie. it is accessible:
![a](https://user-images.githubusercontent.com/10230144/151326350-903021f3-4de6-448d-8fe1-1c492aa2a965.png)

**Reply by diderich** [reply]
After debugging, I found the issue:
![ccc](https://user-images.githubusercontent.com/10230144/151331143-5a807bad-36d8-4fa7-84b4-0b8e0d3e0019.png)

The library used does not allow self-signed certificates in the certification chain. We have set up our own certification authority for our intranet, which leads to this issue.

**Reply by mariuszkrzaczkowski** [reply]
https://github.com/YetiForceCompany/YetiForceCRM/blob/b3990234da7d5addbd37a864f7bd0ee5740d9d9d/app/Utils/ConfReport.php#L670
It should not verify the certificate 
`'verify' => false`

**Reply by diderich** [reply]
To check if the URL exists, the function called is `\App\Fields\File::isExistsUrl()`, rather than `getRequest()` in `ConfReport.php`. Adding the option `'verify' => false` to the `RequestHttp` call in that function, solves the problem.

Thanks for the help.

**Reply by mariuszkrzaczkowski** [reply]
fixed https://github.com/YetiForceCompany/YetiForceCRM/commit/f74dc9bda2d2925dc5d73ee6c5b631edfa83b280
